src/Model/File.php

```diff
@@ -27,6 +27,9 @@ class File extends Model
     /** @var Filesystem */
     public $flysystem;
 
+    /** @int Delay in seconds used to avoid race condition when cleaning up draft records */
+    protected int $cleanupDraftsDelay = 5;
```

Comment:
It can tak much longer time until user submit the form.

Reply:
Set to 60. Anyway user can change it as it wants.

Reply:
it should be like `2 * 24 * 3600`